.tool-versions

```diff
@@ -1,2 +1,2 @@
-erlang 25.0.2
-elixir 1.13.4-otp-25
+elixir 1.14.2-otp-25
```

Comment:
This should be a separate PR.

Its important to keep changes as short as possible and within the same domain